scripts/prepare_ml_data.py

```diff
@@ -175,8 +211,7 @@ def main():
     iterate_over_dataloader_and_write_to_disk(datamodule.test_dataloader(), DST_TEST_PATH)
     _LOG.info("Done!")
 
-    # save configuration to gcs
-    save_configuration_to_cloud(configuration=config, cloud=CLOUD)
+    save_yaml_configuration(config)
 
 
 if __name__ == "__main__":
```

Comment:
nice work making that much nice with the direct paths

Reply:
Oh, I can't take any credit!  `fsspec` does all the magic :)   There's still more work to do to properly use `fsspec` throughout our code though (issue #153 tracks that)